internal/container/deviceservice.go

```diff
@@ -7,14 +7,25 @@
 package container
 
 import (
+	"github.com/edgexfoundry/device-sdk-go/pkg/models"
 	"github.com/edgexfoundry/go-mod-bootstrap/di"
 	contract "github.com/edgexfoundry/go-mod-core-contracts/models"
 )
 
 // DeviceServiceName contains the name of device service implementation in the DIC.
 var DeviceServiceName = di.TypeInstanceToName(contract.DeviceService{})
+var ProtocolDiscoveryName = "ProtocolDiscovery"
```

Comment:
use di.TypeInstanceToName for these also.

Reply:
updated